config/config.zkevm.node.toml

```diff
@@ -0,0 +1,69 @@
+IsTrustedSequencer = true
```

Comment:
This file was introduced to allow us to configure the node as needed by the bridge to test the integration during the Local and E2E tests. What do you suggest to do in this case instead of having this file?

Reply:
Previously this file provided the local configuration and was included inside the docker image, so it was not required. If it was needed modify the properties we were modifying the environment variables in the docker compose file. I sugest follow the same approach

Reply:
Hm, I see... let me try to remove this file and keep the environment running only using ENV vars in the docker-compose file.

Reply:
Done, removed the `node` config file, and defined the configurations via ENV var.